service/crt/service.go

```diff
@@ -29,6 +30,11 @@ const (
 	// resyncPeriod is the period for re-synchronizing the list of objects in k8s
 	// watcher. Set to 1 minute to make the watch more robust.
 	resyncPeriod time.Duration = time.Minute * 1
+
+	TPRName        = "certificate"
```

Comment:
The TPR name should actually come from https://github.com/giantswarm/certificatetpr/blob/989414c916fafbc70d7e09fb48ef053b7290a9e4/name.go#L5, but there is the complete name. Is the interface of the abstraction in operatorkit wrong?